src/applications/vaos/tests/appointment-list/components/PastAppointmentsList/index.unit.spec.jsx

```diff
@@ -503,4 +505,36 @@ describe('VAOS <PastAppointmentsList>', () => {
 
     expect(screen.queryByText(/You don’t have any appointments/i)).not.to.exist;
   });
+
+  describe('Past cancelled appointments', () => {
+    const appointment = getVAAppointmentMock();
+    appointment.attributes = {
+      ...appointment.attributes,
+      startDate: pastDate.format(),
+      vdsAppointments: [
+        {
+          currentStatus: null,
+        },
+      ],
+    };
+
+    it('should not display', async () => {
+      const iterator = CANCELLED_APPOINTMENT_SET.values();
+      let status = iterator.next();
+      while (!status.done) {
```

Comment:
This seems like it might be more complicated that we need here. I think you could pick a couple cancelled statuses to test and that would be good enough.

Reply:
That's interesting. I started out testing for one cancelled status and figured making the test more robust would be better. I can change if you really, really want it.

Reply:
It is definitely more thorough, I was just thinking that it was hard to imagine a scenario where this would catch a bug that testing one or two statuses wouldn't.

You don't need to remove it, it's not wrong. I would say though that you could avid having to deal with the iterator by doing `const cancelledStatuses = Array.from(CANCELLED_APPOINTMENT_SET)`. That will give you a regular array.

Ideally, we'd also work around the eslint issue that was ignored below, but it's probably not critical for a test, since normally these tests would run one after the other anyway.

Reply:
Ok. Modified to test 1 on of the statues.